docs/guide/authentication.md

```diff
@@ -1,5 +1,117 @@
 ---
-title: Authentication 
+title: Authentication
 ---
 
-## Authentication 
+## Intro
+
+In Kubernetes regular end-users are assumed to be managed by an outside,
+independent service. In this regard, Kubernetes does not have objects which
+represent normal user accounts. Regular users cannot be added to a cluster
+through an API call.
+
+API requests are tied to either a normal user or a service account, or are
+treated as anonymous requests. This means every process inside or outside the
+cluster, from a human user typing kubectl on a workstation, to kubelets on
+nodes, to members of the control plane, must authenticate when making requests
+to the API server, or be treated as an anonymous user.
+
+### User-Management
+
+For Kubernikus clusters the user management is handled by OpenStack's Identity
+Service (Keystone). Only users that have been given a `kubernikus_admin` or
+`kubernikus_member` roles by an Keystone administrator are allowed to interact
+with the service or clusters.
+
+### Authentication
+
+The authentication against Kubernikus clusters is based on x509 certificates.
+Encoded into the certificate's `Common Name` field is the user name. The
+certificate's `organization` fields indicate the user's OpenStack role
+assignments.
```

Comment:
We should probably mention that Openstack roles are prefixed with `os:` so `kubernikus_admin` in OpenStack is turned into `os:kubernikus_admin` in k8s.